index.js

```diff
@@ -16,7 +16,7 @@ const noop = () => {};
 
 class Analytics {
   /**
-   * Initialize a new `Analytics` with your Segment project's `writeKey` and an
+   * Initialize a new `Analytics` with your Segment project"s `writeKey` and an
```

Comment:
install eslint, eslint-config-universe, prettier, and @babel/core and let eslint --fix take care of things